docs/metadata-cascade-policies.md

```diff
@@ -0,0 +1,72 @@
+Metadata Cascade Policies
+=========================
+
+A metadata cascade policy indicates if a metadata instance value should be cascaded to files and subfolders in a specific folder.
+
+Any user with edit permisions on a folder can create metadata cascade policies for that given folder. Policies are asigned to exactly one folder and exactly one metadata instance on that folder. It should be noted that there is some delay from file upload to metadata application.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Create Metadata Cascade Policy](#create-metadata-cascade-policy)
+- [Get a Metadata Cascade Policy](#get-a-metadata-cascade-policy)
+- [Get Metadata Cascade Policies for Folder](#get-metadata-cascade-policies-for-folder)
+- [Force Apply Metadata Cascade Policy](#force-apply-metadata-cascade-policy)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Create Metadata Cascade Policy
+------------------------------
+
+To create a new metadata cascade policy, call
+`MetadataCascadePolicyManager.CreateCascadePolicyAsync(String folderId, string scope, string templateKey)`.
```

Comment:
The types in the method signature should use consistent casing; I recommend sticking with "string" for all three